etc/profile-m-z/mediathekview.profile

```diff
@@ -27,6 +29,7 @@ include disable-interpreters.inc
 include disable-programs.inc
```

Comment:
Can we add `include disable-shell.inc` without breaking things? Like the comment below on the other potential includes, just test, and if it breaks mediathekview leave it out.

Reply:
New pr.